recipes/bcolz/meta.yaml

```diff
@@ -1,56 +0,0 @@
-{% set name = "bcolz" %}
```

Comment:
Same here, please restore other recipes to the state they are on master. Thanks!